### PR TITLE
Fix top value when top is not provided

### PR DESCRIPTION
**Pull request by najagasi**
## Description

This PR is to revert the changes that are made in this PR https://github.com/Azure/azure-powershell/commit/dafc0d9cec89f260aebf3e61335d02e4943044f6#diff-5fcd32ad9abc940415b096a7b99c3f3882e4c27dd7806430212796a347c94952
its causing requests made to the RP with top=2147483647 when no $top is requested.

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/PolicyInsights/PolicyInsights/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,9 @@
 -->
 ## Upcoming Release
 
+## Version 1.4.1
+* Undo bug with version 1.4.0
```
Comment (pilor) [inline src/PolicyInsights/PolicyInsights/ChangeLog.md:23]: This isn't descriptive enough.  Say something about reverting a change in 1.4.0 to pull in all pages by default.  Default page size is now 'X' for Get-AzPolicyState

Comment (najagasi) [reply src/PolicyInsights/PolicyInsights/ChangeLog.md:23]: pagination is not yet supported in powershell it is to be implemented, not sure how to word this without using word page.
when a user does not provide top we return 1000 rows, once paging is supported here we will send in 1000 rows plus a nextLink.
the description for this release was support pagination, do i need to instead say "unsupport pagination"

Comment (pilor) [reply src/PolicyInsights/PolicyInsights/ChangeLog.md:23]: Let Liz/Kemley word it :-)  However, you definitely need to mention which cmdlet this affects

Comment (najagasi) [reply src/PolicyInsights/PolicyInsights/ChangeLog.md:23]: Will add "Retracting changes made in powershell that increased request row limit. Removed incorrect statement of supporting paging" as suggested by Kemley

**Comment by msJinLei** [comment]
@najagasi Please do not add new version to changelog and just put new items under upcoming release

**Review by msJinLei — APPROVED** [review]

**Pull request merged**